### DISCUSSION TITLE
Windows binaries

### DISCUSSION THREAD

**Discussion by robertha64** [stronnag/bbl2kml#1] [General]
Poor Windows users binaries ?

**Comment by stronnag** [comment]
What exactly is the problem with the Window's binary?

**Comment by robertha64** [comment]
I can not find it. There is no working link bbl2kml.exe for me at code page.

**Comment by stronnag** [comment]
try "Releases", that's where binaries live.

**Comment by robertha64** [comment]
Sorry find it, I did not look on whole page.

**Comment by synersignart** [comment]
Hi Stronnag I try to run the 
flightlog2kml-1.0.26-win64-x86_64 .exe 
and for some reason on windows 10 it doesnt load up too.
